### PR TITLE
[LICENSE] add ALv2 licenses

### PR DESCRIPTION
**Pull request by szha**
Signed-off-by: Sheng Zha <zhasheng@amazon.com>

### CODE REVIEW

File: applications/nightly_build/test_infogan.py [hunk 1]
```diff
@@ -1,3 +1,5 @@
+# SPDX-License-Identifier: Apache-2.0
+
 ###############################################################################
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License. See License.txt in the project root for
```
Comment (wenbingl) [inline applications/nightly_build/test_infogan.py:5]: The MIT license information is still here, should it be removed as well?

Comment (szha) [reply applications/nightly_build/test_infogan.py:5]: Thanks for the catch. Let me update my script for identifying it.

**Review by wenbingl — APPROVED** [review]

**Comment by szha** [comment]
@wenbingl do you know what caused linux-conda-ci to not have been triggered?

**Comment by wenbingl** [comment]
> @wenbingl do you know what caused linux-conda-ci to not have been triggered?

I just made a pipeline configuration update, can you push a trivial change to trigger the pipeline again?

**Pull request merged**